.docker/README.md

```diff
@@ -0,0 +1,9 @@
+# ROS2 Control Docker Containers
+
+Meant to provide basic docker containers for CI or development purposes.
+
+## Source folder and tag
+Downloads and builds ros2_controls into a workspace for development use exactly as is found [here](https://ros-controls.github.io/control.ros.org/getting_started.html#compiling).
+
+## Release folder and tag
+Installs ros2_control in the base ros2 docker container as mentioned [here](https://ros-controls.github.io/control.ros.org/getting_started.html#getting-started).
```

Comment:
Also... I think it'd be very useful for newcomers or new-first-issue people to work in one of the containers, mounting their local checkout of the repo inside. Do you think that's worth adding too?

Reply:
My recent push added in information on how to get set up with persistent data in a docker container. I think it might be a good idea to reference this section in good-first-issue.md.